authexternalbrowser.go

```diff
@@ -311,10 +310,17 @@ func doAuthenticateByExternalBrowser(
 			buf.Grow(bufSize)
 		}
 		if encodedSamlResponse != "" {
-			httpResponse := buildResponse(application)
-			c.Write(httpResponse.Bytes())
+			httpResponse, err := buildResponse(application)
+			if err != nil && errAccept == nil {
+				errAccept = err
+			}
+			if _, err = c.Write(httpResponse.Bytes()); err != nil {
```

Comment:
Why do we want to overwrite `errAccept` in this case, but not in the previous one? (there is an `errAccept == nil` check in the previous `if`)

Reply:
Very good catch, indeed! Fixed.